sentry-android-core/api/sentry-android-core.api

```diff
@@ -189,24 +189,18 @@ public final class io/sentry/android/core/SentryAndroidOptions : io/sentry/Sentr
 	public fun setProfilingTracesIntervalMillis (I)V
 }
 
-public final class io/sentry/android/core/SentryInitProvider : android/content/ContentProvider {
+public final class io/sentry/android/core/SentryInitProvider {
```

Comment:
I ran `./gradlew apiDump` to update this file. This should be safe and are NOT breaking changes because these APIs are internal and nothing really changed.